identity-account/src/account/account.rs

```diff
@@ -247,54 +282,66 @@ impl Account {
     &self,
     old_state: &IdentityState,
     new_state: &IdentityState,
+    fragment: &Option<String>,
```

Comment:
Opinion: rename `signing_method_query` for consistency?

```suggestion
    signing_method_query: &Option<String>,
```